src/data_source/storage/sql/query/explorer.rs

```diff
@@ -404,32 +404,14 @@ where
                         let time: Option<i64> = row.try_get("time").map_err(|e| QueryError::Error {
                             message: format!("failed to get column time {e}"),
                         })?;
-                        let size: i32 = row.try_get("size").map_err(|e| QueryError::Error {
+                        let size: Option<i32> = row.try_get("size").map_err(|e| QueryError::Error {
```

Comment:
payload.size is a required field in schema so is it ever going to return None? I think we have to do left join above?

Reply:
nvm its nullable

Reply:
I actually this is this probably still a valid concern.

Reply:
we insert header and null payload in insert_leaf() function so no need for left join here